xcodeproj/internal/xcodeprojinfo.bzl

```diff
@@ -297,7 +301,33 @@ def _skip_target(
             "{}:{}".format(package_label_str, ctx.rule.attr.generator_name),
         )
 
+    # Set `alias_labels` only for raw `test_suite` rules by checking for the
+    # existence of the `generator_name` attribute. This attribute will be set
+    # in `ios_*_test_suite` rules.
+    if ctx.rule.kind == "test_suite" and not getattr(ctx.rule.attr, "generator_name"):
```

Comment:
This is very specific. We only want this for `test_suite` and `generator_name` will be set for `ios_*_test_suite` rules and we don't need to alias labels in that scenario but the rule kind is still `test_suite`. So need a way to differentiate raw `test_suite` and `ios_*_test_suite` here.

Reply:
Why do we care if we do this for `ios_*_test_suite`? Is this just an optimization?

Reply:
Applying the `alias_labels` logic for `ios_*_test_suite` targets used in custom schemes breaks things, since it will try to expand to the runners and for `ios_*_test_suite` what we actually want is to keep the parent `test_suite` as the scheme in Xcode building the test bundle and letting Xcode be the runner.